.gitignore

```diff
@@ -71,3 +71,7 @@ wasms/*.wasm.gz
 .canbench_output
 .nx/cache
 /artifacts
+
+# The latest memory dump of the station will change every time the tests are run due to 
+# the UUID being generated with a random seed. We don't need to track this file.
```

Comment:
> the UUID being generated with a random seed

the seed is derived deterministically in tests (it doesn't depend on system time etc.)

Reply:
What makes the memory non-deterministic are the time-stamps stored in various records which are derived from the current time of PocketIC which is set to now: https://github.com/dfinity/orbit/blob/ecd210dbca70e69235462a102c30d4a40bf74309/tests/integration/src/setup.rs#L95